modules/linear_boltzmann_solvers/lbs_solver/lbs_structs.h

```diff
@@ -230,6 +229,8 @@ struct Options
   bool verbose_inner_iterations = true;
   bool verbose_ags_iterations = false;
   bool verbose_outer_iterations = true;
+  int max_ags_iterations = 1;
```

Comment:
Were we going to change this default value after the discussion last Wednesday?

Reply:
Let me look at this a bit. I can default it to something like 10 which may be a better option.

Reply:
I'm going to change the default to 10 iterations. This is going to change the answers for some of the regression tests. I'll make sure that each test that this affects converges the upscattering to a decent value.